src/main/kotlin/kweb/html/HtmlDocumentSupplier.kt

```diff
@@ -44,6 +48,10 @@ internal object HtmlDocumentSupplier {
         }
     }
 
+    fun generateId() : String {
```

Comment:
Should this be private? Not sure where it's being used, but may belong in util/misc.kt.

Reply:
It's currently only being used in this file, so I should have made it private. We may need to move the id generation somewhere else though, so this function might get moved.